[Bug] [dlink-catalog-mysql-1.16] META-INF/services folder not created

### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dlink/issues?q=is%3Aissue) and found no similar issues.


### What happened

META-INF/services folder not created

error:
Caused by: org.apache.flink.table.api.ValidationException: Could not find any factory for identifier 'dlink_mysql' that implements 'org.apache.flink.table.factories.CatalogFactory' in the classpath.

Available factory identifiers are:

generic_in_memory
        at org.apache.flink.table.factories.FactoryUtil.discoverFactory(FactoryUtil.java:545)
        at org.apache.flink.table.factories.FactoryUtil.getCatalogFactory(FactoryUtil.java:723)
        at org.apache.flink.table.factories.FactoryUtil.createCatalog(FactoryUtil.java:420)
        ... 79 more

![image](https://user-images.githubusercontent.com/18652779/207033581-9565899a-2733-4079-9282-56f98b2904f1.png)


### What you expected to happen

dlink-0.7/dev etc
META-INF/services folder not created

error:
Caused by: org.apache.flink.table.api.ValidationException: Could not find any factory for identifier 'dlink_mysql' that implements 'org.apache.flink.table.factories.CatalogFactory' in the classpath.

Available factory identifiers are:

generic_in_memory
        at org.apache.flink.table.factories.FactoryUtil.discoverFactory(FactoryUtil.java:545)
        at org.apache.flink.table.factories.FactoryUtil.getCatalogFactory(FactoryUtil.java:723)
        at org.apache.flink.table.factories.FactoryUtil.createCatalog(FactoryUtil.java:420)
        ... 79 more

![image](https://user-images.githubusercontent.com/18652779/207033581-9565899a-2733-4079-9282-56f98b2904f1.png)


### How to reproduce

no 

### Anything else

no

### Version

0.7.0

### Are you willing to submit PR?

- [ ] No I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dlink-catalog/dlink-catalog-mysql/dlink-catalog-mysql-1.16/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory
+++ b/dlink-catalog/dlink-catalog-mysql/dlink-catalog-mysql-1.16/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory
@@ -1,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+com.dlink.flink.catalog.factory.DlinkMysqlCatalogFactory

